misc/git-tag-maybe.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+base=5.9
+cal=$(date +%Y%m%d)
+chicken=0
+tag="v${base}.${cal}.${chicken}"
+
+r=0
+
+desc=$(git describe --tags --always)
+case "${desc}" in
+	v*)
```

Comment:
This may not work as expected.
If once `v5.9.20201007.0` was tagged and some commits were added, `git describe --tags --always` would return `v5.9.20201007.0-N-gHHHHHHH`. (`N` is a number of commits from the last tag, `HHHHHHH` is the commit ID.)
If `--exact-match` option is specified, `git describe` will fail if the current commit is not tagged.

Reply:
Thank you. I tried --exact-match but it seems to me that the optoin doesn't work expectedly:
```
$ git describe --tags --always
9ce35430
$ git tag x
$ git describe --exact-match  
fatal: no tag exactly matches '9ce35430b0866d055fb18090d85bd00c1dea43d0'
$ echo $?
128
$ git describe --tags --always
x
```

How about pattern 
```
   v*.0)
``` 
instead of `v*` ?
Too stingy?

Reply:
How about `git describe --tags --exact-match`?